joint_limits:
  base_axis_x:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  base_axis_y:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_1:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_2:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_3:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_4:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_5:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_6:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  lwr_joint_7:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  joint_before_finger1:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0
  joint_before_finger2:
    has_velocity_limits: true
    max_velocity: 1.6
    has_acceleration_limits: false
    max_acceleration: 0